### ISSUE TITLE
Fix OWLAPI version number in output

### ISSUE THREAD

**Issue by jamesaoverton** [ontodev/robot#411]
Hilmar reports seeing `<!-- Generated by the OWL API (version 1.2.0) https://github.com/owlcs/owlapi -->` (the ROBOT version) instead of the expected `<!-- Generated by the OWL API (version 4.5.6) https://github.com/owlcs/owlapi -->`.

**Comment by jamesaoverton** [comment]
@hlapp Did you see this when using ROBOT as a command-line tool or as a library?

**Comment by matentzn** [comment]
I just updated all pipelines to ROBOT 1.2.0 (release), and I get, for all generated artefacts, this diff:

```
- <!-- Generated by the OWL API (version 4.2.6) https://github.com/owlcs/owlapi -->
+ <!-- Generated by the OWL API (version 1.2.0) https://github.com/owlcs/owlapi -->
```

Its also a bit funny that the given OWLAPI version corresponds exactly to the ROBOT release version.

**Comment by jamesaoverton** [comment]
@matentzn The problem is just that the way OWLAPI checks its version number is finding the ROBOT version number instead. Probably here:

https://github.com/owlcs/owlapi/blob/version4/api/src/main/java/org/semanticweb/owlapi/util/VersionInfo.java#L31

It's not a deep problem.

The only weird thing is that I'm seeing the correct OWLAPI version most of the time.

**Comment by rctauber** [comment]
@matentzn - I am having trouble recreating this problem locally. What operating system are you using for this? And can you provide me the steps that you followed to see this output? Thank you!

**Comment by matentzn** [comment]
Give me a second, I may have been still on 1.2.0-alpha.

**Comment by matentzn** [comment]
Well, still same. v.1.2.0

go to odk, build this: https://github.com/INCATools/ontology-development-kit/blob/master/docker/odkfull/Dockerfile

and run a robot command from in there. (see [here](https://github.com/INCATools/ontology-development-kit/blob/master/template/src/ontology/run.sh) how to run it).

I know the build worked because the remove command is successfully executed.

Check [here](https://github.com/obophenotype/xenopus-phenotype-ontology/pull/12/files) an example were it happened.

**Comment by hlapp** [comment]
> Did you see this when using ROBOT as a command-line tool or as a library?

Command line. I haven't used it as a library yet.

**Comment by hlapp** [comment]
> I am having trouble recreating this problem locally. What operating system are you using for this?

Download the release robot.jar, download robot shell script, run `robot convert` such that the output is RDF/XML.

The funny thing is that you can't reproduce this with a locally built robot.jar (at least I don't have this problem if I build the robot.jar myself). You have to be using the release robot.jar to see the issue.

**Comment by jamesaoverton** [comment]
I believe that this problem was limited to the JAR I attached to the GitHub release of 1.2.0. This should be resolved in the 1.3.0 release I just made.

**Comment by hlapp** [comment]
Indeed I can confirm that this fixes it. Issue can be considered fixed and closed as far as I am concerned.
